protected/humhub/modules/activity/messages/ru/base.php

```diff
@@ -1,29 +1,29 @@
-<?php
+<?php /* Translated by Paul (https://paul.bid) www.paul.bid@gmail.com */
 return array (
-  '<strong>E-Mail</strong> Summaries' => '<strong>Уведомления</strong> по email',
+  '<strong>E-Mail</strong> Summaries' => '<strong>Email Уведомления</strong>',
```

Comment:
Okay, well that's more a matter of style.

Reply:
I think we should try to keep the strong style as original text when it is possible.
Yes, sometimes a translated text has a diffrent order of the words, so it may looks strange.
But please note such strong styled text is used for headers usually like this:
![strong-style](https://github.com/user-attachments/assets/1dd64b36-294d-4c78-99ae-22f6bb59d53a)

Reply:
@hide-me I think such questions should be addressed to the authors and designers of this project, because, in my opinion, the strong style should be as closely related to the original text as possible, other it looks like an attempt to change the design.

@luke- What do you think?

Reply:
Difficult topic. We want to revise that anyway.
If it doesn't make sense from a wording point of view, the strong tag is also ok over the entire title.

Reply:
Nice, do I can keep it! It was last unsolved moment.
Finally, I can start to upload changed files!